docs/Reference/Errors.md

```diff
@@ -128,45 +217,68 @@ fastify.listen({ port: 3000 }, function (err, address) {
 })
 ```
 
-#### FST_ERR_NOT_FOUND
-<a id="FST_ERR_NOT_FOUND"></a>
-
-404 Not Found.
-
-#### FST_ERR_OPTIONS_NOT_OBJ
-<a id="FST_ERR_OPTIONS_NOT_OBJ"></a>
-
-Fastify options must be an object.
-
-#### FST_ERR_QSP_NOT_FN
-<a id="FST_ERR_QSP_NOT_FN"></a>
-
-QueryStringParser option should be a function.
-
-#### FST_ERR_SCHEMA_CONTROLLER_BUCKET_OPT_NOT_FN
-<a id="FST_ERR_SCHEMA_CONTROLLER_BUCKET_OPT_NOT_FN"></a>
-
-SchemaController.bucket option should be a function.
-
-#### FST_ERR_SCHEMA_ERROR_FORMATTER_NOT_FN
-<a id="FST_ERR_SCHEMA_ERROR_FORMATTER_NOT_FN"></a>
-
-SchemaErrorFormatter option should be a non async function.
-
-#### FST_ERR_AJV_CUSTOM_OPTIONS_OPT_NOT_OBJ
-<a id="FST_ERR_AJV_CUSTOM_OPTIONS_OPT_NOT_OBJ"></a>
-
-ajv.customOptions option should be an object.
-
-#### FST_ERR_AJV_CUSTOM_OPTIONS_OPT_NOT_ARR
-<a id="FST_ERR_AJV_CUSTOM_OPTIONS_OPT_NOT_ARR"></a>
-
-ajv.plugins option should be an array.
-
-#### FST_ERR_VERSION_CONSTRAINT_NOT_STR
-<a id="FST_ERR_VERSION_CONSTRAINT_NOT_STR"></a>
-
-Version constraint should be a string.
+Below is a table with all the error codes that Fastify uses.
+
+<table>
+<thead>
```

Comment:
I'm not a big fan of Markdown table syntax, but I think we should use it.

```md
| Code | Description | How to solve | Discussion |
|---|---|---|---|
| a | b | c | d|
```

| Code | Description | How to solve | Discussion |
|---|---|---|---|
| a | b | c | d|